pkg/tfvars/baremetal/baremetal.go

```diff
@@ -87,6 +90,28 @@ func TFVars(numControlPlaneReplicas int64, libvirtURI, apiVIP, imageCacheIP, boo
 		driverInfo["deploy_kernel"] = fmt.Sprintf("http://%s/images/ironic-python-agent.kernel", net.JoinHostPort(imageCacheIP, "80"))
 		driverInfo["deploy_ramdisk"] = fmt.Sprintf("http://%s/images/ironic-python-agent.initramfs", net.JoinHostPort(imageCacheIP, "80"))
 
+		var hostFile *asset.File
```

Comment:
I'm not sure this is the standard way directly using `asset.File` in here. Because that brings about Marshalling&Unmarshalling like below.

Reply:
Thanks for your review, I understand your concerns. I'm not sure if this is a standard way either, but I saw [cases](https://github.com/openshift/installer/blob/master/pkg/asset/cluster/tfvars.go#L157) where Unmarshalling is used to process `asset.File data`. And currently the configuration of RAID and BISO can only be obtained from `mastersAsset.HostFiles`. It seems that using Marshalling&Unmarshalling is a relatively straightforward way.

Reply:
Unmarshalling the asset file is appropriate.

Reply:
I would take it a step further and suggest that you should be getting *all* of the configuration for the host from the asset file instead of from the install config. The install config is used as input into the asset file. But after the masters asset is created, the host files become the source of truth. The user is permitted to modify the host files on-disk, and the installer should respect those changes.

Reply:
> The user is permitted to modify the host files on-disk, and the installer should respect those changes.

@staebler Thanks for your review. This is indeed reasonable. Considering that this is to modify the existing implementation, I will open a new PR to do what you suggested in the future.

Reply:
@staebler We hope that this feature can be merged into OCP 4.10, could we implement the improvements you mentioned later?

Reply:
Yes, I did not mean to suggest that any changes needed to be made to this PR. I was only supporting the notion that reading the asset files is acceptable, and in many ways recommended.